CppCoreGuidelines.md

```diff
@@ -7984,34 +7984,7 @@ Avoid resource leaks.
 
 ### <a name="Rh-make_unique"></a>C.150: Use `make_unique()` to construct objects owned by `unique_ptr`s
 
-##### Reason
-
-`make_unique` gives a more concise statement of the construction.
-It also ensures exception safety in complex expressions.
-
-##### Example
-
-    unique_ptr<Foo> p {new Foo{7}};    // OK: but repetitive
-
-    auto q = make_unique<Foo>(7);      // Better: no repetition of Foo
-
-    // Not exception-safe: the compiler might interleave the computations of arguments as follows:
-    //
-    // 1. allocate memory for Foo,
-    // 2. construct Foo,
-    // 3. call bar,
-    // 4. construct unique_ptr<Foo>.
-    //
-    // If bar throws, Foo will not be destroyed, and the memory-allocated for it will leak.
-    f(unique_ptr<Foo>(new Foo()), bar());
-
-    // Exception-safe: calls to functions are never interleaved.
-    f(make_unique<Foo>(), bar());
-
-##### Enforcement
-
-* Flag the repetitive usage of template specialization list `<Foo>`
-* Flag variables declared to be `unique_ptr<Foo>`
```

Comment:
I'm choosing the R.23 enforcement over the C.150 enforcement because R.23's appears to be more controversial (as noted in the issue for R.22 vs C.151)